crypto.go

```diff
@@ -38,9 +38,12 @@ const (
 	// The UUIDv5 is derived from a fixed namespace UUIDv4
 	// "794253a4-310b-449d-9d8d-4575e8923f40" and a version string
 	// "TFSP1;ED25519;BLAKE2B"
-	ToznyFieldSignatureVersionV1 = "e7737e7c-1637-511e-8bab-93c4f3e26fd9" // UUIDv5 TFSP1;ED25519;BLAKE2B
-	FILE_VERSION                 = 3
-	FILE_BLOCK_SIZE              = 65536
+	ToznyFieldSignatureVersionV1         = "e7737e7c-1637-511e-8bab-93c4f3e26fd9" // UUIDv5 TFSP1;ED25519;BLAKE2B
+	FILE_VERSION                         = 3
+	FILE_BLOCK_SIZE                      = 65536
```

Comment:
now in this repo, since there is some more consistency than in e3db-go, please use CamelCase for constants